src/generic/GenericTypes.jl

```diff
@@ -1006,11 +1006,9 @@ end
 
 const RationalFunctionFieldDict = CacheDictType{Tuple{Field, Union{Symbol, Vector{Symbol}}}, Field}()
 
-mutable struct RationalFunctionFieldElem{T <: FieldElement, U <: Union{PolyRingElem, MPolyRingElem}} <: AbstractAlgebra.FieldElem
+struct RationalFunctionFieldElem{T <: FieldElement, U <: Union{PolyRingElem, MPolyRingElem}} <: AbstractAlgebra.FieldElem
    d::FracFieldElem{U}
    parent::RationalFunctionField{T, U}
-
-   RationalFunctionFieldElem{T, U}(f::FracFieldElem{U}) where {T <: FieldElement, U <: Union{PolyRingElem, MPolyRingElem}} = new{T, U}(f)
```

Comment:
Removing this constructor is technically a breaking change. But I am still in favor of this PR

Reply:
I am fine with marking this PR as breaking *shrug*

Reply:
We could also just keep it. It is just useless.